internal/tiltfile/tiltfile_test.go

```diff
@@ -1706,7 +1708,7 @@ func TestExtraImageLocationOneImage(t *testing.T) {
 	f.dockerfile("env/Dockerfile")
 	f.dockerfile("builder/Dockerfile")
 	f.file("Tiltfile", `k8s_yaml('crd.yaml')
-k8s_kind('Environment', image_json_path='{.spec.runtime.image}')
+k8s_image_json_path(kind='Environment', path='{.spec.runtime.image}')
```

Comment:
nit: you should probably have `path` be the first arg (and be an arg not a kwarg) if that's how we expect users to do it.